AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -1101,7 +1115,16 @@ - (void)applicationWillEnterForeground {
   }
 }
 
-- (void)applicationDidEnterBackground {
+/**
+ * Clear currently running SFAuthenticationSession.
+ */
+- (void)clearAuthenticationSession API_AVAILABLE(ios(11)) {
```

Comment:
If my feedback is right there is no need to have a method to do cancellation and strong reference breaking both at the same time but we can keep the method for isolating the complexity of the iOS version dependent code to cancel only the session.

Reply:
I think we still don't need this method since we don't need to explicitly break the strong reference with `nil` instead we just cancel before reassigning the new session.

Reply:
_For info at first I thought cancelling the session programmatically would call the completion handler but it doesn't._